src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -105,13 +100,36 @@ public void save(AddressBook addressBook) throws StorageOperationException {
             marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             marshaller.marshal(toSave, fileWriter);
 
+            // sets file to read-only
+            //File file = new File(DEFAULT_STORAGE_FILEPATH);
+            //file.setReadOnly();
+
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code, if not it will clutter up the codebase in the long run :)